index.d.ts

```diff
@@ -1994,12 +2004,21 @@ export namespace stream {
 
 		class WorksheetReader {
 			constructor(options: WorksheetReaderOptions);
+			name: string;
+			state: string;
```

Comment:
I could not find this field in a workbook reader. Could I please add a permalink into the source code?